### PR TITLE
Add course detail section 

### PR DESCRIPTION
**Pull request by adarsh-technocrat**
**Describe the changes you've made**

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

**Additional context (OPTIONAL)**

**Test plan (OPTIONAL)**

A good test plan should give instructions that someone else can easily follow.

**Checklist**

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

**Provide a Deployed link of route/page that needs to review**
Preview: Deploy preview link here with the appropriate route

<!-- Please add deployed link to here text >

### CODE REVIEW

**Comment by adarsh-technocrat** [comment]
@Abhishek-kumar09  ping !!

**Review by Abhijay007 — COMMENTED** [review]
LGTM 👍

File: src/components/CourseDetailsComponent/AboutCourse.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { Box, Container, makeStyles } from "@material-ui/core";
+import LearningInfo from "./SubComponents/AboutElements/LearningInfo";
+import CourseIncludes from "./SubComponents/AboutElements/CourseIncludes";
+import DescriptionContainer from "./SubComponents/AboutElements/DescriptionContainer";
+import CourseSchedule from "./SubComponents/AboutElements/CourseSchedule";
+import SkillsSection from "./SubComponents/AboutElements/SkillsSection";
+import Prerequisites from "./SubComponents/AboutElements/Prerequisites";
+import StudentCourseReview from "./SubComponents/AboutElements/StudentCourseReview";
+function AboutCourse() {
```
Comment (Abhishek-kumar09) [inline src/components/CourseDetailsComponent/AboutCourse.js:10]: A line break

File: src/components/CourseDetailsComponent/SubComponents/AboutElements/LearningInfo.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  Box,
+  Grid,
+  makeStyles,
+  Typography,
+  ListItem,
+  ListItemAvatar,
+  Avatar,
+  ListItemText,
+} from "@material-ui/core";
+
```
Comment (Abhishek-kumar09) [inline src/components/CourseDetailsComponent/SubComponents/AboutElements/LearningInfo.js:12]: remove line break

File: src/components/CourseDetailsComponent/SubComponents/AboutElements/LearningInfo.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  return (
+    <Box className={classes.root}>
+      <Box className={classes.container}>
+        <Typography variant="h4" color="textSecondary" gutterBottom>
+          What you will learn
+        </Typography>
+        <Box mt={2} mb={2}>
+          <Grid container item xs={12} spacing={2}>
+            {listElement.map((items, index) => (
+              <>
```
Comment (Abhishek-kumar09) [inline src/components/CourseDetailsComponent/SubComponents/AboutElements/LearningInfo.js:33]: Whenever you are mapping something make sure to provide keys.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Some Nits,
Also, please add the mock server call wherever necessary.

File: src/components/CourseDetailsComponent/AboutCourse.js (hunk 1 above)
Comment (adarsh-technocrat) [reply src/components/CourseDetailsComponent/AboutCourse.js:10]: Sure @Abhishek-kumar09  !!

File: src/components/CourseDetailsComponent/SubComponents/AboutElements/LearningInfo.js (hunk 3 above)
Comment (adarsh-technocrat) [reply src/components/CourseDetailsComponent/SubComponents/AboutElements/LearningInfo.js:33]: Sure @Abhishek-kumar09   :)

**Comment by adarsh-technocrat** [comment]
Hey, @Abhishek-kumar09  please review this PR, and about mock data, I'll implement that in the existing draft PR!

**Review by Abhishek-kumar09 — APPROVED** [review]

**Pull request merged**